doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -38,7 +38,10 @@
 <a name="4.5"></a>
 ## 4.5 CNI plugins
 
-> This chapter should describe which CNI plugins are part of the Rerefence Architecture.
+The used CNI multiplexer/metapulgin must be [DANM](https://github.com/nokia/danm) with [Calico](https://github.com/projectcalico/cni-plugin) as the CNI what complies with the basic networking assumptions of Kubernetes.
```

Comment:
Question: Do we need to impose the use of **Calico** and **DANM** via this **MUST** clause ?
There are other options as well like Multus+Calico, Cilium, CNI-Genie etc.
Any criteria for shortlisting of DANM and calico specifically ?

Reply:
So I'm not averse to specifying but as @msidana suggests, we need to be super clear as to the rationale of the choice (i.e. which RM and RA requirements are only met by this choice and no others).

Reply:
@msidana I think it is the purpose of this chapter to make component level choices. 
@tomkivlin Sure I can add some more detailed reasoning, however #693  for Chapter 03 covers most of them.

Reply:
@msidana are you ok with the changes to resolve this conversation?

Reply:
Yes @tomkivlin, based on https://github.com/cntt-n/CNTT/pull/821#issuecomment-572572859, I am resolving this conversation. Thanks.